src/applications/mhv-landing-page/containers/App.jsx

```diff
@@ -35,6 +35,7 @@ const App = () => {
   const unreadMessageAriaLabel = resolveUnreadMessageAriaLabel(
     unreadMessageCount,
   );
+  const mhvAccount = useSelector(state => state.mhvAccount);
```

Comment:
Maybe change this to be a boolean with some other name like `hasMhvAccount` or something like that. It will make the intent clearer.

Reply:
thank you, will do